rubicon_ml/repository/base.py

```diff
@@ -296,8 +298,10 @@ def _archive(
         -------
         filepath of newly created archive
         """
+        remote_s3 = True if remote_rubicon_root and remote_rubicon_root.startswith("s3") else False
         if remote_rubicon_root is not None:
             archive_dir = os.path.join(remote_rubicon_root, slugify(project_name), "archives")
+
```

Comment:
can we similarly do `root_dir = remote_rubicon_root if remote_rubicon_root is not None else self.root_dir` then just have the `archive_dir = ...` line once?